documentation/2_Guiding_Principles.md

```diff
@@ -396,3 +396,72 @@ Such roles originate from regulations, like GDPR or DGA:
 * Personal data Intermediary are not described in the IDSA Rulebook.
 * Clearing house service providers are described in the IDS RAM and subject of the obverservability capability in the functional requirements section of the IDSA Rulebook.
 * Marketplace and other value creation services are covered in the IDSA Rulebook above and in the optional capabilites section of the functional requirements.
+
+# Understanding Roles and Layers in Data Spaces
+
+Data spaces are multi-layered ecosystems that rely on the seamless integration of technical protocols, business processes, and legal frameworks. One of the foundational challenges in governing data spaces lies in the consistent definition and use of key concepts such as **"roles," "policies,"** and **"contracts."** These terms often carry different meanings across domains. This chapter establishes a clear separation between technical and non-technical interpretations of such concepts to support the development of interoperable and trustworthy data spaces.
+
+## Layers of a Data Space
+
+Data spaces can be structured into three primary layers, each serving distinct functions:
+
+1. **Technical Layer** – Encompasses the architecture and protocols (e.g., Data Space Protocol (DSP), Decentralized Claims Protocol (DCP)) that facilitate secure and interoperable data exchanges.
+2. **Business/Operational Layer** – Manages the services, interactions, and workflows that enable value generation and marketplace activity.
```

Comment:
The figure uses the term economic layer and we should have it consistent. So, either use business layer or economic layer. Operational layer doesn't fit IMO.